triggers/service/BUILD.bazel

```diff
@@ -69,6 +69,19 @@ da_scala_binary(
     ],
 )
 
+# The ref-ledger-authentication binary that we exercise from the
+# service tests.
+java_import(
```

Comment:
Note that as soon as you start relying on this you also need to bundle it in the SDK.

Reply:
Let me make sure that fact is not forgotten. But let me check, the intent is only to exercise it from the tests. Do we still need to bundle it in the SDK?

Reply:
Issue https://github.com/digital-asset/daml/issues/6486

Reply:
I don’t understand why you only need this in the tests. Surely anyone that wants to run the trigger service with auth also needs this? Are you saying it’s up to them to install it and set it up?

Reply:
That was the plan. The idea was to document "if you have a REST service available satisfying this interface <link to specification> then pass these configuration parameters e.g. `--with-auth-url` (and whatever else) and trigger service will call it. if you don't have one of your own making, you are welcome to use this one <link to ref-ledger-authentication URL>" sort of thing.

Reply:
and so in that way, we integrate auth without locking down any specific auth provider, test it, satisfy the dabl requirements and avoid having to release a new SDK component.

Reply:
Making it configurable sounds sensible but it might still make sense to ship the auth service as a default implementation so users can get started quickly.

Reply:
sounds good. i'll leave https://github.com/digital-asset/daml/issues/6486 open as is accordingly.